global.json

```diff
@@ -1,9 +1,9 @@
 {
     "tools": {
-        "dotnet": "9.0.100",
+        "dotnet": "9.0.102",
         "runtimes": {
             "dotnet/x64": [
-                "8.0.11"
```

Comment:
I see the project still targets `net8.0`. At what cadence do we move the `tools` and `sdk` sections to new runtime or SDK versions? I'm confused on how all of these relate for Sign CLI and the desired development environment.

Reply:
Good point.  Because Sign CLI is a tool, `tools` affect Sign CLI itself when run from this directory.  I've updated it to use 8.x and added a comment.

SDK versions can be later versions.  We've aimed to use the latest .NET release.

Reply:
Ah, now I remember.  `tools.dotnet` and `tools.runtimes` are out of sync because Sign CLI depends on the .NET 8 runtime, but the .NET 8 version of `xcopy-msbuild` has CG alerts, which are only resolved by updating to .NET 9 SDK for tools.  See https://github.com/dotnet/msbuild/commit/977f965b0e18488a0768da26277177524f6d1107